R/lineage.R

```diff
@@ -358,21 +358,21 @@ ipg2lin <- function(accessions, ipg_file,
     if (length(refseq_rows) != 0) {
         refseq_ipg_dt <- ipg_dt[refseq_rows, ]
         refseq_lins <- GCA2lin(refseq_ipg_dt,
-            assembly_path = refseq_assembly_path,
+            .data$assembly_path = refseq_assembly_path,
             lineagelookup_path
         )
     }
     if (length(genbank_rows) != 0) {
         genbank_ipg_dt <- ipg_dt[genbank_rows, ]
-        genbank_lins <- GCA2lin(gca_ipg_dt,
-            assembly_path = genbank_assembly_path,
+        genbank_lins <- GCA2lin(.data$gca_ipg_dt,
+            .data$assembly_path = genbank_assembly_path,
```

Comment:
The `.data` prefix is not required in this instance, as `gca_ipg_dt` and `assembly_path` are an arguments to `GCA2lin()`.